pkgs/build-support/fetchsvnssh/default.nix

```diff
@@ -1,4 +1,4 @@
-{stdenvNoCC, subversion, sshSupport ? false, openssh ? null, expect}: 
+{stdenvNoCC, subversion, sshSupport ? true, openssh ? null, expect}:
```

Comment:
Shoud this `openssh ? null` become just `openssh` if the local default for ssh support is `true`?

Reply:
In theory: no preference, in practice: it usually it is not in most other expressions.

Reply:
Strictly speaking, the default it will get depends on `callPackage` implementation used, so the above should be okay, normally. In this instance, however, the output will depend on `openssh` anyway because the path gets burned into derivation irrespective of `sshSupport`. I'm not sure if fixing this should be in scope of this PR, though.

Reply:
I was only asking about default argument values, I agree with both your implied claims (ignoring the argument if the support if off is good, but fixing this is out of scope).

I think if it is called in Nixpkgs exactly once, it is acceptable to assume this is the only type of `callPackage` that will be applied.